Cargo.toml

```diff
@@ -10,5 +10,5 @@ build = "build.rs"
 [dependencies]
 libc = "*"
```

Comment:
[According to crates.io documentation](http://doc.crates.io/crates-io.html#using-cratesio-based-crates), "0.2.6" is actually equivalent to "^0.2.6" and does not pin it to exactly that version, but everything "0.2.x" with x > 6. Just learned that with the 1.6 release.

I'm personally in favour of explicitly adding the ^ requirement like you did to avoid any confusion, but as crates.io displays it without ^, I thought others would maybe want to leave it out.

Regarding ^0.2 vs ^0.2.6, I think it's best to use the last released patch level as this is exactly what we're developing against, and freeze in when an actual releaser are done of qmlrs to stay compatible with whatever will be the current state then.